03-viz.Rmd

```diff
@@ -290,6 +290,151 @@ faithful_scatter <- ggplot(faithful, aes(x = waiting, y = eruptions)) +
 faithful_scatter
 ```
 
+
+### The Canadian languages data set
+
+We will return to the `can_lang` data set, which contains counts of languages from the 2016 Canadian census.
+
+**Question:** Is there a relationship between the number of people who speak a language as their mother tongue and the number of people who speak that language as their primary spoken language at home? And is there a pattern in the strength of this relationship in the higher-level language categories (Official languages, Aboriginal languages, or non-official and non-Aboriginal languages)?
+
+```{r 03-canlang-example, message = F, warning = F}
+can_lang <- read_csv("data/can_lang.csv")
+can_lang
+```
+
+We will begin with a scatter plot of the `mother_tongue` and `most_at_home` columns from our data frame.
+
+```{r mother-tongue-vs-most-at-home, fig.width=5.75, fig.height=5, warning=FALSE, fig.cap = "Scatter plot of number of Canadians reporting a language as their mother tongue vs the primary language at home"}
+ggplot(can_lang, aes(x = most_at_home, y = mother_tongue)) +
+  geom_point()
+``` 
+To improve the interpretability of the data visualization in Figure \@ref(fig:mother-tongue-vs-most-at-home) we should replace the default axis names with a more informative labels. 
+
+```{r mother-tongue-vs-most-at-home-labs, fig.width=5.75, fig.height=5, warning=FALSE, fig.cap = "Scatter plot of number of Canadians reporting a language as their mother tongue vs the primary language at home with x and y labels"}
+ggplot(can_lang, aes(x = most_at_home, y = mother_tongue)) +
+  geom_point() +
+  xlab("Language spoken most at home (number of Canadian residents)") +
+  ylab("Mother tongue (number of Canadian residents)")
+```
+
+Most of the data points from the 214 observations in this data set are bunched up in the lower left-handside of the visualization in Figure \@ref(fig:mother-tongue-vs-most-at-home-labs). The data is clumped because many more people in Canada speak the two languages represented by the points in the upper right corner. By filtering the data, we can confirm these two points correspond to the two official languages (English and French):
+
+```{r english-mother-tongue}
+can_lang %>%
+  filter(language == "English" | language == "French")
+```
+> `|` is a logical operator representing "OR", which allows us to filter the rows where the language column is equal to "English" OR equal to "French". 
+
+To answer our question, we will need to adjust the scale of the x and y axes so that they are on a log scale:
```

Comment:
We should explain here how a log scale will solve our problem of the bunched up data points, as well as the gotcha's associated with using it (in particular, values of 0...)